java/core/src/test/java/com/whylogs/core/metrics/TestMetric.java

```diff
@@ -0,0 +1,21 @@
+package com.whylogs.core.metrics;
+
+import java.util.ArrayList;
+import org.junit.Assert;
+import org.testng.annotations.Test;
+
+@Test
+public class TestMetric {
+
+  @Test
+  public void testMetrics() {
+    ArrayList<Metric> metrics = new ArrayList<>();
```

Comment:
It's not clear to me what this is testing

Reply:
This was just double checking that this is working as expected in a Collection of Metrics with the merge currently having that odd way of making sure it's an IntegralMetric and it won't fail